coordinator/src/main/java/org/apache/uniffle/coordinator/QuotaManager.java

```diff
@@ -53,13 +53,14 @@ public class QuotaManager {
   private final Map<String, Integer> defaultUserApps = Maps.newConcurrentMap();
 
   public QuotaManager(CoordinatorConf conf) {
-    this.quotaFilePath = conf.get(CoordinatorConf.COORDINATOR_QUOTA_DEFAULT_PATH);;
+    this.quotaFilePath = conf.get(CoordinatorConf.COORDINATOR_QUOTA_DEFAULT_PATH);
     final Long updateTime = conf.get(CoordinatorConf.COORDINATOR_QUOTA_UPDATE_INTERVAL);
     try {
       hadoopFileSystem = HadoopFilesystemProvider.getFilesystem(new Path(quotaFilePath), new Configuration());
     } catch (Exception e) {
       LOG.error("Cannot init remoteFS on path : " + quotaFilePath, e);
     }
+    LOG.warn("We will update the quota of users regularly.");
```

Comment:
Why we need this?

Reply:
This is to prompt the user that QuotaManager starts normally.

Reply:
How about use LOG.info?

Reply:
Updated.